app/views/step_by_step_pages/index.html.erb

```diff
@@ -28,56 +28,83 @@
 <div class="govuk-grid-row govuk-!-margin-top-5">
   <div class="govuk-grid-column-one-quarter">
     <div class="govuk-grid-row">
-      <div class="govuk-grid-column-full">
-        <%= render "govuk_publishing_components/components/input", {
-          label: {
-            text: "Title or base path"
-          },
-          name: "search",
-          id: "filterTableInput"
-        } %>
-      </div>
-      <div class="govuk-grid-column-full">
-        <%= render "govuk_publishing_components/components/select", {
-          id: "filterStatus",
-          label: "Status filter",
-          full_width: true,
-          options: [
-            {
-              text: "All",
-              data_attributes: {
-                show: "all"
-              },
-              selected: true
-            },
-            {
-              text: "Draft",
-              data_attributes: {
-                show: "draft"
-              }
-            },
-            {
-              text: "Live",
-              data_attributes: {
-                show: "live"
-              }
-            },
-            {
-              text: "Scheduled",
-              data_attributes: {
-                show: "scheduled"
-              }
+      <%= form_tag({}, enforce_utf8: false, method: :get) do %>
+        <div class="govuk-grid-column-full">
+          <%= render "govuk_publishing_components/components/input", {
+            label: {
+              text: "Title or URL",
+              bold: true
             },
-            {
-              text: "Unpublished changes",
-              data_attributes: {
-                show: "unpublished_changes"
+            name: "title_or_url",
+            value: params[:title_or_url],
+            id: "title_or_url"
+          } %>
+        </div>
+        <div class="govuk-grid-column-full">
+          <%= render "govuk_publishing_components/components/label", {
+            text: "Status",
+            html_for: "status",
+            bold: true
+          } %>
+          <%= render "govuk_publishing_components/components/select", {
+            id: "status",
+            name: "status",
+            label: "",
```

Comment:
Are we not specifying `label` here because it doesn't come out as bold, so we're defining our own label separately?

I wonder if we can just remove the `label: ""` altogether.

Reply:
Yes. "label" in the select component is not bold. I tried removing, but the component doesn't render properly, so I think it's a required field.